src/ResourceManagement/NotificationHubs/NotificationHubs.Tests/Tests/ScenarioTests.NamespaceTests.CRUDAuthorizationRules.cs

```diff
@@ -27,6 +27,7 @@ namespace NotificationHubs.Tests.ScenarioTests
     using System.Net;
     using System.Collections.Generic;
     using System.Linq;
+
     public partial class ScenarioTests 
     {
         [Fact]
```

Comment:
Spoke to @shahabhijeet offline but the reason why I kept a single test for each CRUD sets was to avoid repetitiveness. If I split CRUD for a test I have a generic problem : 

1. To test Create call : I need to do a Get call to verify the Create and need to do a Delete for cleanup. 
2. To test Update call : I need to do a Create first and do multiple Gets to verify create and update. Do a delete in the end to clean up
3. To test Get : Need to do a Create anyways and do a Delete to clean up

So it made sense to have one test which does all